flink-runtime-web/src/main/java/org/apache/flink/runtime/webmonitor/WebRuntimeMonitor.java

```diff
@@ -424,6 +425,33 @@ protected void initChannel(SocketChannel ch) {
 		LOG.info("Web frontend listening at " + address + ':' + port);
 	}
 
+	public static Archiver[] getArchivers() {
```

Comment:
I see... changing that will probably be an unrelated refactoring. So let's leave it as is and add a comment. I think it won't even hurt to add a note why it is a static method etc.